<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientFormDialog</class>
 <widget class="QDialog" name="PatientFormDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>402</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>544</width>
    <height>402</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>544</width>
    <height>402</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PatientFormDialog</string>
  </property>
  <widget class="ComboBox" name="sexCombo">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>М</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ж</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Телефон:</string>
   </property>
  </widget>
  <widget class="LineEdit" name="addressEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>50</number>
   </property>
  </widget>
  <widget class="LineEdit" name="phoneEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>330</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Фамилия:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Име:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Адрес:</string>
   </property>
  </widget>
  <widget class="CityLineEdit" name="cityLineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>68</number>
   </property>
  </widget>
  <widget class="LineEdit" name="HIRBNoEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>8</number>
   </property>
  </widget>
  <widget class="IdLineEdit" name="idLineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
  </widget>
  <widget class="ComboBox" name="typeComboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>59</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <item>
    <property name="text">
     <string>ЕГН</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ЛНЧ</string>
    </property>
   </item>
  </widget>
  <widget class="LineEdit" name="fNameEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>50</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Населено място:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string> Пол:</string>
   </property>
  </widget>
  <widget class="DateLineEdit" name="birthEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="showGroupSeparator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="maximumDateTime">
    <datetime>
     <hour>23</hour>
     <minute>59</minute>
     <second>59</second>
     <year>9999</year>
     <month>12</month>
     <day>30</day>
    </datetime>
   </property>
   <property name="minimumDate">
    <date>
     <year>1900</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
   <property name="date">
    <date>
     <year>1900</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>350</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Рожденна дата:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Презиме:</string>
   </property>
  </widget>
  <widget class="LineEdit" name="lNameEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>50</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>№ на ЗОК:</string>
   </property>
  </widget>
  <widget class="LineEdit" name="mNameEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>50</number>
   </property>
  </widget>
  <widget class="Label" name="codeLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>../uiComponents/qt_derived/lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>CityLineEdit</class>
   <extends>QLineEdit</extends>
   <header>../uiComponents/qt_derived/citylineedit.h</header>
  </customwidget>
  <customwidget>
   <class>DateLineEdit</class>
   <extends>QDateEdit</extends>
   <header>../uiComponents/qt_derived/datelineedit.h</header>
  </customwidget>
  <customwidget>
   <class>IdLineEdit</class>
   <extends>QLineEdit</extends>
   <header>../uiComponents/qt_derived/idlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>../uiComponents/qt_derived/combobox.h</header>
  </customwidget>
  <customwidget>
   <class>Label</class>
   <extends>QLabel</extends>
   <header>../uiComponents/qt_derived/label.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>idLineEdit</tabstop>
  <tabstop>birthEdit</tabstop>
  <tabstop>sexCombo</tabstop>
  <tabstop>fNameEdit</tabstop>
  <tabstop>mNameEdit</tabstop>
  <tabstop>lNameEdit</tabstop>
  <tabstop>cityLineEdit</tabstop>
  <tabstop>HIRBNoEdit</tabstop>
  <tabstop>addressEdit</tabstop>
  <tabstop>phoneEdit</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>typeComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
